src/components/i18n/Outdated.astro

```diff
@@ -0,0 +1,14 @@
+---
+import { useTranslations } from "../../i18n/utils";
+import type { Params } from "../../i18n/utils";
+
+const { lang } = Astro.params as Params;
+const translation = useTranslations(lang);
+---
+
+<section id="i18n-outdated">
+	<h2>{translation("i18n-outdated.title")}</h2>
+	<p class="description">
+		{translation("i18n-outdated.description")}
+	</p>
+</section>
```

Comment:
Something like this might look better to serve as a warning, apart from the content on the website itself:
```suggestion
<div
    class="w-full max-w-4xl bg-yellow-800 bg-opacity-50 border-l-4 border-orange-500 text-orange-100 p-4"
    role="alert"
>
    <p class="font-bold">{translation("i18n-outdated.title")}</p>
    <p>{translation("i18n-outdated.description")}</p>
</div>
```

Not quite sure about the use of `id="i18n-outdated"` currently, but feel free to adapt 😄

Reply:
This looks stands out much better this way, thanks!

Reply:
Definitely much better now!

I noticed you still made the entire banner "prose-like". It'd be great to hear what other reviewers think, but I have two suggestions:

<ol><li>

I'd recommend against the use of headers in banners, because it competes visually with the hero and header. Although the banner should be attention seeking enough to warn users about the outdated nature of the page, it shouldn't steal attention to become one of the "main titles/headings" of the page. It's a little large right now and breaks the visual hierarchy of the page.

  For reference, my suggestion was sourced from: https://v1.tailwindcss.com/components/alerts#left-accent-border

</li><li>

Note that we don't _necessarily_ need `prose` here -- it's mostly for content-like elements, or ones from the output of say a markdown processor. This is entirely up to you, but for a non-content component like this, elements without prose would work just fine -- extra classes means an extra selectors (and hence possibly redundant properties) applied :)

</ol>

Reply:
Ah, no worries!